schema.sql

```diff
@@ -0,0 +1,34 @@
+DROP TABLE IF EXISTS Generation ;
```

Comment:
Is there a reason we didn't use cascade? It doesn't matter, I just include it as standard.